src/components/single-form.js

```diff
@@ -71,65 +71,6 @@ var singleForm = React.createClass({
     }
     return true;
   },
-  renderPaymentOptions: function() {
```

Comment:
Your call, but if it's possible to move these changes into another PR, I could probably get to reviewing just the changes to single-form.js much much faster.

Reply:
can do.